### PR TITLE
feat: add rm support to strigify and uglify-js as package dependencies

### LINKED ISSUE #117: toFixed include RM as parameter
It would be a good addition if we add RM(round mode) as parameter in `toFixed` function same as `round` function.

Feature Request:

`.toFixed(dp, rm) ⇒ string`

### PR DESCRIPTION
**Pull request by nutboltu**
This PR fixes #117 
- Adding `rm` as a second optional parameter for `toExponential`
- Adding `rm` as a second optional parameter for `toFixed`
- Adding `rm` as a second optional parameter for `toPrecision`
- Adding `uglify-js` as devDependencies for build

[x] Docs are updated as per changes.
[ ] Unit tests

### CODE REVIEW

**Comment by nutboltu** [comment]
@MikeMcl would you please take a look? I will add unit tests after getting an initial review from you.

File: big.js [hunk 1]
```diff
@@ -268,7 +269,7 @@
       n = k - x.e;
 
       // Round?
-      if (x.c.length > ++k) round(x, n, Big.RM);
+      if (x.c.length > ++k) round(x, n, rm || Big.RM);
```
Comment (MikeMcl) [inline big.js:272]: Needs to be 
```js
if (x.c.length > ++k) round(x, n, rm === UNDEFINED ? Big.RM : rm);
```
as `rm` may be `0`.

**Review by MikeMcl — CHANGES_REQUESTED** [review]
I was about to reply this very evening anyway. Honest!

I've actually already done this work - the code part anyway - but, as usual, one change leads to another, and I have a bit more refactoring to do before pushing.

I'll take this PR from you anyway, though, as it includes other stuff. When I publish, I will probably remove the minified version and all that goes with it.

Thanks for your good work.

**Comment by nutboltu** [comment]
@MikeMcl no worries. Thanks for your time to review. Feel free to take my changes. Whenever you are done with the changes, I'll close the PR.
However, if you want me PR to get merge first and then you'll start working on top of it, that would be another option. Let me know which works for you.

File: big.js (hunk 1 above)
Comment (nutboltu) [reply big.js]: fixed.

**Pull request merged**